### DISCUSSION TITLE
Slack channel

### DISCUSSION THREAD

**Discussion by unbreakabl3** [vmware/build-tools-for-vmware-aria#201] [Q&A] [answered]
Hello,
there is no Slack channel anymore. Are there any plans to create a new one or move it to another platform (Discord or... ) ?

**Comment by Michaelpalacce** [comment, marked as answer]
@unbreakabl3 we are currently discussing alternatives for Slack, stay tuned.

**Reply by unbreakabl3** [reply]
Hey, @Michaelpalacce .
Do you have any updates on that maybe?

Thanks.

**Comment by Michaelpalacce** [comment]
@VenelinBakalov Can you chime in?

**Comment by VenelinBakalov** [comment]
Hi @unbreakabl3 , for now there is no available alternative, so both for that reason and for promoting the usage of Github itself, we agreed to start using Github Issues and Discussions (based on the kind of topic that needs to be resolved). 
It will take some learning curve both for our time and for contributors outside of the organisation but we believe that in the long run it will bring greater benefit to use Github as a central point of communication around Build Tools for Aria.
So feel free to open any topics here and in Issues (if an issue is later found to be a configuration topic and not actual code change, it can be transitioned into a discussion) and don't hesitate to tag us, so that in case we miss the thread, we will receive an email about it :) 
Converting Issue to Discussion option can be found on the right side of the menu:
![image](https://github.com/vmware/build-tools-for-vmware-aria/assets/22296648/957e16a8-b22d-4d62-85e7-167162ee810e)

**Comment by unbreakabl3** [comment]
Hey, @VenelinBakalov .
Thank you for that reply. I will happily start to use it. 
Just to confirm: "_...benefit to use Gitlab..._" - Gitlab is a typo, right? :)

**Comment by VenelinBakalov** [comment]
Hahah fair point, that's what happens when you work with too many tools at the same time, let me update the original comment :)
